redaxo/src/core/lib/login/login.php

```diff
@@ -566,15 +561,7 @@ public static function startSession()
                 session_save_path((string) $sessionConfig[$env]['save_path']);
             }
 
-            $cookieParams = static::getCookieParams();
-
-            session_set_cookie_params(
-                $cookieParams['lifetime'],
-                $cookieParams['path'],
-                $cookieParams['domain'],
-                $cookieParams['secure'],
-                $cookieParams['httponly'],
-            );
+            session_set_cookie_params(static::getCookieParams());
```

Comment:
Inzwischen kann man die Cookie-Params als Array übergeben. Und da wird dann auch `samesite` unterstützt.
Somit brauchen wir unsere eigene Implementierung nicht mehr mit dem Cookie-Rewriting, denke ich.

Reply:
zwischenzeitlich gibt es einen "partitioned" parameter im http, den php-src noch nicht kennt.

d.h. das gleiche problem, nur mit einem neuen parameter

https://github.com/php/php-src/issues/12646

Reply:
Noch nicht so richtig mit beschäftigt. Haben wir da denn akuten Bedarf bei unserem Session-Cookie?

Reply:
ich habe mich mit diesem konkret auch noch nicht beschäftigt.

 mir ging es eher darum, dass es in letzter zeit immer wieder mal neue attribute gibt und in php-src in der regel diese erst einige jahre später eingeführt werden

Reply:
Wobei ich das dann trotzdem so erstmal reinnehmen würde, denke ich. Denn für samesite macht die Logik ja keinen wirklichen Sinn mehr. 
Falls wir mal wieder ähnliches brauchen, können wir es uns aus der History holen, denke ich.

Reply:
Ok